### PR TITLE
Updated the WorkloadType accepted values

### PR DESCRIPTION
**Pull request by HarvestingClouds**
Updated the WorkloadType accepted values in the table to be consistent with the bullet points. Removed the "SAPHanaDatabase" as the accepted value. Confirmed that in the module version 4.8.0 (latest) of Az.RecoveryServices module this value is not yet available.

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/RecoveryServices/RecoveryServices/help/Get-AzRecoveryServicesBackupItem.md [hunk 1]
```diff
@@ -286,7 +286,7 @@ Workload type of the resource. The acceptable values for this parameter are:
 Type: Microsoft.Azure.Commands.RecoveryServices.Backup.Cmdlets.Models.WorkloadType
 Parameter Sets: GetItemsForContainer, GetItemsForVault
 Aliases:
-Accepted values: AzureVM, AzureFiles, MSSQL, FileFolder, SAPHanaDatabase
+Accepted values: AzureVM, AzureFiles, MSSQL, FileFolder
```
Comment (wyunchi-ms) [inline src/RecoveryServices/RecoveryServices/help/Get-AzRecoveryServicesBackupItem.md:289]: `SAPHanaDatabase` is still accept. Please submit a issue if you meet problem with `SAPHanaDatabase`.

Comment (VeryEarly) [inline src/RecoveryServices/RecoveryServices/help/Get-AzRecoveryServicesBackupItem.md:289]: this value is supported from client side: 
https://github.com/Azure/azure-powershell/blob/main/src/RecoveryServices/RecoveryServices.Backup/Cmdlets/Item/GetAzureRmRecoveryServicesBackupItem.cs#L101

Please submit an [issue](https://github.com/Azure/azure-powershell/issues/new?assignees=&labels=needs-triage%2Cbug&template=1-AZ-BUG-REPORT.yml) and describe if you have further questions.

Comment (HarvestingClouds) [reply src/RecoveryServices/RecoveryServices/help/Get-AzRecoveryServicesBackupItem.md:289]: I believe it was there in the AzureRm module and not in the Az module. If you have the link to the Az module's repo that would be very helpful. I tested again and it doesn't list as a valid value on the client-side. 

Still, if this should be opened as an issue then I believe the bullet above should match the list of accepted values for the Parameter WorkloadType. I opened PR #17041 for the same. Please review it here: https://github.com/Azure/azure-powershell/pull/17041

Comment (VeryEarly) [reply src/RecoveryServices/RecoveryServices/help/Get-AzRecoveryServicesBackupItem.md:289]: > 

The above link is for the Az module. I think developer forgot to update related docs.

Comment (hiaga) [reply src/RecoveryServices/RecoveryServices/help/Get-AzRecoveryServicesBackupItem.md:289]: @HarvestingClouds : Get-AzRecoveryServicesBackupItem cmdlet now support fetching the backup items for SAPHANADatabase WorkloadType. Are you facing any issues with the same in latest 5.1.0 Az.RecoveryServices module ?

**Comment by VeryEarly** [comment]
close this one and merge #17041

**Pull request closed without merging**